README.md

```diff
@@ -13,53 +13,60 @@ import '@internetarchive/collection-browser';
 </collection-browser>
 ```
 
-## Local Demo with `web-dev-server`
+## Local Demo with `vite`
 ```bash
-yarn start
```

Comment:
Question: Why are we changing from yarn to npm?

Reply:
i figured if we can use npm vs yarn now on our projects, but can change it back to yarn

Reply:
@dualcnhq Re. "why?" 
I was looking for an answer like, "We're trying to standardize all our JS projects on ..."
I remember the question of 'npm vs yarn" being asked in a recent meeting -- but I don't remember a firm decision being made. 
Do at least @iisa and @traceypooh agree on a direction?

Reply:
let's move back to yarn to stay steady until release.  Vite looks compatible w yarn